keyboards/mitosis/mitosis.h

```diff
@@ -1,51 +1,41 @@
-#ifndef MITOSIS_H
-#define MITOSIS_H
-
+#pragma once
 #include "quantum.h"
 
-#define red_led_off   PORTF |= (1<<5)
-#define red_led_on    PORTF &= ~(1<<5)
-#define blu_led_off   PORTF |= (1<<4)
-#define blu_led_on    PORTF &= ~(1<<4)
-#define grn_led_off   PORTD |= (1<<1)
-#define grn_led_on    PORTD &= ~(1<<1)
-
-#define set_led_off     red_led_off; grn_led_off; blu_led_off
-#define set_led_red     red_led_on;  grn_led_off; blu_led_off
-#define set_led_blue    red_led_off; grn_led_off; blu_led_on
-#define set_led_green   red_led_off; grn_led_on;  blu_led_off
-#define set_led_yellow  red_led_on;  grn_led_on;  blu_led_off
-#define set_led_magenta red_led_on;  grn_led_off; blu_led_on
-#define set_led_cyan    red_led_off; grn_led_on;  blu_led_on
-#define set_led_white   red_led_on;  grn_led_on;  blu_led_on
+/* port,pin assignments for LED indicators */
+#define led_red   F5
+#ifdef MITOSIS_DATAGROK_I2CHACK
```

Comment:
Is this flag intended for your personal board?
If so, it would be nicer to follow our existing pattern of:
```c
#ifndef led_green
#    define led_green   F1
#endif
```

Where you add `#define led_green D5` to the keymap level.

Reply:
It is. Fixed in latest commit.